gprotorch/data_featuriser/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .featurisation import (
```

Comment:
@bojana-rankovic @leojklarner @a-r-j @henrymoss I'm guessing we should rename the gprotorch directory to gauche at some point. Not sure where we should do that and how difficult it's going to be.

Reply:
Should be quire straightforward. Can probably just do a regex over the codebase.

Reply:
Yes, I'll take care of that after we merge this PR.